packages/flutter_tools/doc/daemon.md

```diff
@@ -76,23 +76,7 @@ It is up to the client to decide how best to display the message; for some clien
 
 ### app domain
 
-#### app.start
-
-The `start()` command is used to start applications.
-
-- `deviceId`: The device to launch the app on; this is required.
-- `projectDirectory`: The project directory; this is required. It is used to determine the application to start.
-- `startPaused`: Start the VM in a paused mode.
-- `route`: A string; the route to use when restoring the application.
-- `mode`: One of either `debug`, `profile`, or `release`.
-- `target`: Optional; the target file to start.
-- `hot`: Optional; whether to start the application using `--hot` mode
-
-On success, returns a map with the fields:
-- `appId`: this is is used when sending app events, and can be used by clients to stop the app (`app.stop`).
-- `deviceId`
-- `directory`
-- `supportsRestart`
+The app domain is only available via `flutter run --machine` and not in `flutter daemon`.
```

Comment:
I would strike this, and only remove the `start` (`and `discover`) commands.

Reply:
Ah, I assumed we couldn't do `restart`/`discover` if the app wasn't launched within this process. If that's not the case then this was a bit overkill, I'll put them back!